app/controllers/ViewTransfersController.scala

```diff
@@ -62,111 +46,136 @@ class ViewTransfersController @Inject() (val consignmentService: ConsignmentServ
 
-  // scalastyle:off cyclomatic.complexity
-  private def contactUsAction(consignmentRef: String): UserAction = {
-    UserAction(contactUs, s"mailto:%s?subject=Ref: $consignmentRef - Issue With Transfer", contactUs)
-  }
+  private def createView(edges: Option[Edges]): Option[ConsignmentTransfers] =
+    edges.map { edge =>
+      val userAction: UserAction = toUserAction(edge.node)
 
-  private def convertStandardStatusesToUserActions(statuses: CurrentStatus, consignmentId: UUID, consignmentRef: String, consignmentType: String): UserAction = {
-    val pageNameToUrlMap: Map[String, String] = standardTransferPageNameToUrl(consignmentId)
+      ConsignmentTransfers(
+        edge.node.consignmentid,
+        edge.node.consignmentReference,
+        userAction.transferStatus,
+        statusColours(userAction.transferStatus),
+        userAction,
+        edge.node.exportDatetime.map(_.format(formatter)).getOrElse("N/A"),
+        edge.node.createdDatetime.map(_.format(formatter)).getOrElse(""),
+        edge.node.totalFiles
+      )
+    }
 
-    statuses match {
-      case s if s.series.isEmpty => UserAction(transferStatusInProgress, pageNameToUrlMap("series"), resumeTransfer)
-      case s if s.series.contains(completedStep) && s.transferAgreement.isEmpty =>
-        UserAction(transferStatusInProgress, pageNameToUrlMap("taPrivateBeta"), resumeTransfer)
+  private def toUserAction(consignment: Node): UserAction = {
+    val judgmentType = consignment.consignmentType.contains("judgment")
+    val consignmentId = consignment.consignmentid.get
+    val consignmentRef = consignment.consignmentReference
+    val statuses = consignment.consignmentStatuses
 
-      case s if s.transferAgreement.contains(inProgressStep)                    => UserAction(transferStatusInProgress, pageNameToUrlMap("taCompliance"), resumeTransfer)
-      case s if s.transferAgreement.contains(completedStep) && s.upload.isEmpty => UserAction(transferStatusInProgress, pageNameToUrlMap("upload"), resumeTransfer)
+    val statusesToCheck: List[ConsignmentStatuses] = if (judgmentType) {
+      statuses.filterNonJudgmentStatuses
+    } else {
+      statuses
+    }
 
-      case s if s.confirmTransfer.isEmpty                                       => convertUploadAndFileCheckStatusesToUserActions(s, pageNameToUrlMap, consignmentRef)
-      case s if s.confirmTransfer.contains(completedStep) && s.`export`.isEmpty => UserAction(transferStatusInProgress, pageNameToUrlMap("fileChecksResults"), resumeTransfer)
-      case s if s.`export`.nonEmpty                                             => convertExportStatusesToUserActions(s, pageNameToUrlMap, consignmentRef, consignmentType)
+    statusesToCheck match {
+      case s if s.containsStatus(ExportType) => toExportAction(s.find(_.statusType == ExportType.id).get, judgmentType, consignmentId, consignmentRef)
+      case s if s.containsStatus(ConfirmTransferType) && s.statusValue(ConfirmTransferType).contains(CompletedValue.value) =>
```

Comment:
Again, not sure if this will make things more complicated to read but you don't need to check `containsStatus(ConfirmTransferType)` first. `s.statusValue(ConfirmTransferType).contains(CompletedValue.value)` will return false if the list doesn't have that status so you can remove it. 
Same applies to the Series check on line 86.

Reply:
That's does help. Thanks